src/FieldDescription/FilterTypeGuesser.php

```diff
@@ -30,6 +30,9 @@
 
 final class FilterTypeGuesser implements TypeGuesserInterface
 {
+    /**
+     * @psalm-suppress DeprecatedConstant
```

Comment:
Maybe we should list the constant deprecated and when to remove them ?

Reply:
I've added comments to these constants